client/post-editor/editor-drawer/index.jsx

```diff
@@ -139,7 +139,10 @@ const EditorDrawer = React.createClass( {
 
 	renderSharing: function() {
 		return (
-			<EditorSharingAccordion
+			<CodeSplitRender
+				require={ function( callback ) {
+					require( [ 'post-editor/editor-sharing/accordion' ], callback );
```

Comment:
My only concern is readability of the tree if we start having loader wrappers everywhere. Nothing jumps out at me immediately as possible improvements, though.